docker-fedora/README.md

```diff
@@ -0,0 +1,54 @@
+# container-engine
+
+This is the container engine system container for fedora systems based on the moby project.
+For more information refer to: https://github.com/moby/moby
+
+Note this only provides the *daemon process*, and does not include CLI tools. The current use
+case for this would be switching between different versions of the engine by updating the
+container.
+
+## Building the image from source:
+
+```
+# git clone https://github.com/projectatomic/atomic-system-containers
+# cd atomic-system-containers/docker-fedora
+# docker build -t container-engine .
+```
+
+## Running the system container, with the atomic CLI:
+
+Pull from registry into ostree:
+
+```
+# atomic pull --storage ostree $REGISTRY/container-engine
+```
+
+Or alternatively, pull from local docker:
+
+```
+# atomic pull --storage ostree docker:container-engine:latest
+```
+
+Install the container:
+
+```
+# atomic install --system --name=container-engine ($REGISTRY)/container-engine
```

Comment:
yeah let's enforce that until https://github.com/projectatomic/atomic/issues/981 is sorted out